docs/developer/app-store/apps/invoices.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Invoices
+sidebar_position: 4
+---
+
+# Invoices
+
+## Metadata
+
+- Saleor version: 3.10
+- Roadmap https://github.com/saleor/apps/labels/App%3A%20Invoices
```

Comment:
```suggestion
- [Roadmap](https://github.com/saleor/apps/labels/App%3A%20Invoices)
```

Reply:
Thats WIP, I will try to adda react component

Reply:
I added but I'm going to ask for a design